app/assets/stylesheets/article-show.scss

```diff
@@ -31,6 +31,97 @@
   height: 10px;
 }
 
+.mod-actions-menu {
+  position: fixed;
+  top: var(--header-height);
+  right: 0;
+  height: calc(100vh - 56px);
+  z-index: 200;
+  transform: translateX(360px);
+  transition: var(--transition-props);
+  &.showing {
+    transform: initial;
+  }
+
+  iframe {
+    height: 100%;
+    border: none;
+    box-shadow: -4px 0px 15px rgba(0, 0, 0, 0.1);
+    width: 360px;
+  }
+}
+
+.mod-actions-menu-btn {
+  position: fixed;
+  bottom: 30px;
+  right: 30px;
+  z-index: 201;
+  cursor: pointer;
+  transition: 0.3s all ease;
+  background-color: var(--base-90);
```

Comment:
We do have quite a lot of styles to choose from for buttons. I know we do not have the color that is defined here, but I think we could use one of the existing colors here so A) you don't have to worry about styling and B) I think we this code you may have some issues on other themes, no?

cc @lisasy